src/compoz.ts

```diff
@@ -342,8 +398,19 @@ export class Compoz {
 		}
 
 		this.elInput.onkeydown = e => {
-			if ((e.ctrlKey || e.metaKey) && e.keyCode === this.keyCodeZ) {
-				document.execCommand("undo")
+			if (e.ctrlKey || e.metaKey) {
+				if (!e.shiftKey && e.keyCode === this.keyCodeZ) {
+					e.preventDefault()
+					this.undo()
```

Comment:
Add `return` statement for each `if` condition to immediately end the function.

Reply:
I am sorry forgot to add the ```return``` statement.